### PR TITLE
Jump menu expanded

### PR DESCRIPTION
**Pull request by discodavey**

### CODE REVIEW

File: test/viewselector.spec.js [hunk 1]
```diff
@@ -329,23 +305,6 @@ describe('A ViewSelector Component', function () {
 
     });
 
-    it('maintains a list of top level article section headings', () => {
```
Comment (NuclearRedeye) [inline test/viewselector.spec.js:332]: We might want to raise a follow up ticket just to ensure that this test is covered somewhere else. If the other tests are only checking if the menu has the default options and not the section headings links then it is possible we might miss a break further down the line.

**Review by NuclearRedeye — APPROVED** [review]
I've made one comment, that I think we should investigate as part of some follow up work. I'm still annoyed that I cannot get your changes to run in a local instance (not your fault!), but I've tested them on the AWS instance and am happy that they are working as expected and no regressions have been introduced.

**Comment by discodavey** [comment]
@NuclearRedeye So you are happy for me to put this live then?

**Comment by NuclearRedeye** [comment]
@discodavey Yep

**Pull request merged**